addons/modules/fnc_getPosFromString.sqf

```diff
@@ -0,0 +1,43 @@
+/* ----------------------------------------------------------------------------
+Function: CBA_fnc_getPosFromString
+
+Description:
+    A function used to get the position of an item when passed as a string. Designed
```

Comment:
Separate the first sentence (short one-line description) from the rest with an empty line, eg
```
A function used to...

Designed...
```